### PR TITLE
Spring Cleaning

### PR DESCRIPTION
**Pull request by patrick-lloyd-ray**
I'd like to tidy up the main readme for this repo before end of Q1. If you have any documentation, information, or suggestions, please make changes here (or direct me to what you would like included).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,4 +23,53 @@ Initialize the packages using:
...
+## Status Board
+
+Here are the BICAN LinkML knowledgebase schemas and their statuses.
+
+| Model | Version | Release | Status |
+|:--|:--|:--|:--|
+| [BICAN BioLink] | [] |  [] | under development |
+| [CCN2] | [] | [] | under development |
+| [KB Model] | [] | [] | under development |
+| [Purple Boxes] | [] | [] | under development |
```
Comment (satra) [inline README.md:37]: perhaps we should call this something else, and make everything under KB Model.

**Review by satra — APPROVED** [review]
in general looks good to me. i think this calls for a renaming of the models to their smaller partitions (genes, transcriptomics, spatial-transcriptomics, transcriptomic-atlas, etc.,.

**Comment by djarecka** [comment]
@satra -should we remove everything what was related to "purple boxes" from "kb-model" and name it "genomics" or "gene" and add everything to the one `kb-model` yaml file just as imports to generate the documentation?

@patrick-lloyd-ray - I've also updated "About" section to include link to the documentation hosted on github website

**Review by djarecka — APPROVED** [review]

**Comment by satra** [comment]
> should we remove everything what was related to "purple boxes" from "kb-model" and name it "genomics" or "gene" and add everything to the one kb-model yaml file just as imports to generate the documentation?

sounds good to me. unless there is an easy way to generate an aggregate doc while keeping models separate.

File: README.md (hunk 1 above)
Comment (patrick-lloyd-ray) [reply README.md:37]: I'll change the name here for now. We can change the path or remove this when we make a release.

**Pull request merged**